lib/osf-components/addon/components/branded-header/component.ts

```diff
@@ -23,7 +22,6 @@ export default class BrandedHeader extends Component<InputArgs> {
     @requiredAction onSearch!: (value: string) => void;
     @tracked showingHelp = false;
 
-    providerModel?: ProviderModel;
```

Comment:
This doesn't seem related to adding new tests. This would have been a good addition to the "Changes" section of the PR, to let reviewers know that this is expected and what's going on here. Do you not have a providerModel any more?